flake.nix

```diff
@@ -25,56 +25,59 @@
         inputs.process-compose-flake.flakeModule
         inputs.cargo-doc-live.flakeModule
         (inputs.leptos-fullstack + /nix/flake-module.nix)
+        ./nix/e2e.nix
       ];
-      perSystem = { config, self', pkgs, lib, system, ... }: {
-        _module.args.pkgs = import inputs.nixpkgs {
-          inherit system;
-          overlays = [
-            inputs.rust-overlay.overlays.default
-          ];
-        };
+      perSystem = { config, self', pkgs, lib, system, ... }:
+        {
```

Comment:
If you move this `{` to the line above (which is how it is in `main`), it will fix the indentation change that has been making the diff harder to read.

Reply:
I see